lib/internal/modules/package_json_reader.js

```diff
@@ -38,4 +38,14 @@ function read(jsonPath) {
   return result;
 }
 
-module.exports = { read };
+/**
+ * 
+ * @param {string} jsonPath 
+ * @param {string} srcPath 
+ */
```

Comment:
Does the linter accept the jsdoc style comments? If so, yay. I'd like to see us adopt this across the board throughout.

Reply:
it doesn't complain about them